web/src/connectors/task-annotator-connector/task-annotator-context.tsx

```diff
@@ -82,7 +82,7 @@ import { useDocumentDataFullLoading } from './use-document-data-full-loading';
 
 type WithFunctionStub<T> = T | (() => void);
 
-type ContextValue = SplitValidationValue &
+export type ContextValue = SplitValidationValue &
```

Comment:
Do you think a more specific name to this interface would go well?

Reply:
I think, yes. Just didn't rename it to not touch this file a lot in this PR. We can update it in the following PRs